data/meson.build

```diff
@@ -0,0 +1,47 @@
+desktop = custom_target(
+    'desktop',
+    input: 'xed.desktop.in.in',
+    output: 'xed.desktop',
+    command: [intltool_merge, '-d', '-u', po_dir, '@INPUT@', '@OUTPUT@'],
+    install: true,
+    install_dir: desktopdir,
+)
+
+appdata = custom_target(
+    'appdata',
+    input: 'xed.appdata.xml.in',
+    output: 'xed.appdata.xml',
+    command: [intltool_merge, '-x', '-u', po_dir, '@INPUT@', '@OUTPUT@'],
+    install: true,
+    install_dir: join_paths(datadir, 'appdata'),
```

Comment:
appdata is the old location, metainfo should be used instead.

https://wiki.debian.org/AppStream/Guidelines#General

Reply:
Again, that's where the autotools version was putting it, but I can fix it easily enough.